doc/tutorials/modeling-complex-events-ctim.md

```diff
@@ -0,0 +1,77 @@
+## Modeling Complex Events in CTIM
+
+CTIM is capable of modeling arbitrarily complex event data. That is, events that are triggered by other events, which themselves might be triggered by other events.
+
+## Synopsis
+
+- Arbitrarily complex cybersecurity events can be modeled in CTIM.
+- Pattern matching or detection rules are modeled with the `indicator` entity.
+- An `indicator` entity can be `based-on` other `indicator` entities.
+- When modeling `indicator` entities, less is more.
+  - Don't include the patterns you match against in your `indicator` entities. They'll be used for detection evasion.
+  - The `indicator` entity should only summarize the threat. Using other entities for describing threats in depth.
+- When a pattern matching or detection rule described by an `indicator` is triggered, we create a `sighting` that is a `sighting-of` relationship to that `indicator`.
+- A `sighting` entity can be `based-on` other `sighting` entities, in addition to being a `sighting-of` an `indicator`.
+
+## Indicators as Detection or Pattern Matching Rules
+
+There are two basic indicator types. This tutorial concerns itself with the first type, which is a detection or pattern matching rule. These rules look for patterns in data, and can run inside of rule engines, expert systems, firewalls, log monitors, or any other kind of scanning or monitoring system.
```

Comment:
`There are two basic indicator types.` -> `There are two basic indicator types: either a detection or pattern matching rule, or a ??`.

Reply:
I'll link to other docs on this one. I don't want to discuss the other type (the observable watch list) in this tutorial.